src/App.tsx

```diff
@@ -1,55 +1,87 @@
-import React from 'react';
+import debounce from 'lodash.debounce';
```

Comment:
You did not commit updated package.json file - it might some troubles.
Also my recomendation try to create own debounce function